.github/workflows/test.yml

```diff
@@ -189,3 +189,23 @@ jobs:
         run: npm run test:local:node:lokijs
       - name: Stop Consumer API
         run: docker compose -f ./backbone/docker-compose/docker-compose.test.yml down
+  integration-test:
+    name: Run Integration Tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Install script dependencies
+        run: npm install --prefix ./.ci
+      - name: Setup dotnet
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: "7.0.x"
+      - name: Build project
+        run: dotnet restore "Backbone.sln" && dotnet build  /property:WarningLevel=0 --no-restore "Backbone.sln"
```

Comment:
Why do you restore and build the solution? This should be done by the Dockerfiles during docker compose up.

Btw, you should probably use Dockerfile instead of Dockerfile.debug in case of the Consumer API

Reply:
That's something I hadn't done initially, but it turns out that building the solution inside Docker makes it impossible to run the tests on the host machine later. See https://github.com/nmshd/backbone/actions/runs/5530508228/jobs/10090204253#step:7:12